_examples/beh-2015-sequencing.md

```diff
@@ -108,24 +109,25 @@ These scenarios are representative of a continuum in future conditions from the
 </p>
 <p> Step 2: Sensitivity analysis. 
 <ul>
-<li> The Sobol's method was used to evaluate the global sensitivity of the objectives to uncertain variables. In addition, the reliability of the supply and the vulnerability of the demand to supply shortfall is estimated. </li> The analysis is repeated 20 times based on 20 sequences of 40 years daily stochastic rainfall sequences and the results of the sensitivity analysis are presented as average values over the 20 sequences. </li>
-<li> Optimal sequence plans' performance to uncertain variables is presented as box-and-whiskers plots displaying the variation of the avarage values for the PV costs, PV GHG emissions, reliability and vulnerability (figure 3) 
+<li> Sobol's method was used for senstivity analysis
+<li> The performance measures were the objectives, reliability (a measure of how often supply exceeds or equals demand) and vulnerability (a measure of demand shortfall). </li> 
+<li> The sensitivity analysis was repeated 20 times using stochastic daily rainfall data for the 40 year planning timeframe. The results were averaged and presented as box-and-whisker plots (figure 3).
```

Comment:
It's not actually the case that the results of the sensitivity analysis was averaged - it's the performance of the plan.